homeassistant/components/rachio/switch.py

```diff
@@ -169,15 +166,19 @@ class RachioZone(RachioSwitch):
     def __init__(self, person, controller, data, current_schedule):
         """Initialize a new Rachio Zone."""
         self._id = data[KEY_ID]
+        _LOGGER.debug("zone_data: %s", data)
```

Comment:
We try to avoid side effects, like logging, in init method.

Reply:
Will fix in a followup once other submitters get their changes in so I don't create any additional conflict pain for them